CONTRIBUTING.md

```diff
@@ -35,32 +35,39 @@ into three categories:
 Remember, if you are unsure about anything, don't hesitate to comment on issues
 and ask for clarifications!
 
-## Pre-commit checks
+## Installation for Development
 
-Numba-CUDA uses [`pre-commit`](https://pre-commit.com/) to run a number of style
-checks in CI. To ensure your contribution will pass the checks, you can also set
-up pre-commit locally to run the checks prior to committing.
+There are some dependencies that are required for developing Numba-CUDA, but are not required for installation or distribution.
+These dependencies are listed under the `dev-cu11` and `dev-cu12` optional dependency groups in our project configuration.
 
-To set up pre-commit with Numba-CUDA, run
+To install Numba-CUDA for development, run this in the root of the repository:
 
+```shell
+pip install -e ".[dev-cu11]"
 ```
-pip install pre-commit
-```
-
 or
-
+```sh
+pip install -e ".[dev-cu12]"
 ```
+
+Or, if you are using conda, run this in the root of the repository:
+```shell
 conda install pre-commit
 ```
```

Comment:
Can we move this below the sentences below related to `pre-commit`. This doesn't need to be run in the root of the repo.

Reply:
Thank you again for the review!